tools/worker-runner/cfg/workerconfig.go

```diff
@@ -102,6 +102,10 @@ func merge(v1, v2 interface{}) interface{} {
 	arr1, arr1ok := v1.([]interface{})
 	arr2, arr2ok := v2.([]interface{})
 	if arr1ok && arr2ok {
+		// check for overflw
+		if len(arr1) > 2*1024*1024*1024 || len(arr2) > 2*1024*1024*1024 {
```

Comment:
Glad we still support merging 1 billion item lists, I'm sure the system will have no problems at that scale